docs/how-to/redeployment.md

```diff
@@ -0,0 +1,37 @@
+# Redeployment Guide
+
+This guide provides the steps necessary for migrating an existing WordPress
+instance to a new charm instance.
+
+## Migrate Database
+
+Follow the instructions
+in [the mysql charm migration guide](https://charmhub.io/mysql/docs/h-migrate-cluster-via-restore)
+to migrate the content of the WordPress MySQL database.
+
+## Migrate Media Files
+
+### Media Files Stored in Kubernetes Storage
+
+If your media files are stored in Kubernetes
+storage (`wp_plugin_openstack-objectstorage_config` is not configured), use the
+following steps to migrate your files:
+
+1. Use the `kubectl cp` command to transfer files from
```

Comment:
juju scp wouldnt work?

Like:
https://discourse.canonical.com/t/increase-storage-used-by-a-charm-workaround/2760/5?u=amandahla

Reply:
Yes, juju scp would be a better choice here, thanks!